plugins/module_utils/routeros.py

```diff
@@ -9,6 +9,7 @@
 from ansible.module_utils.common.text.converters import to_native
 from ansible.module_utils.basic import env_fallback
 from ansible_collections.ansible.netcommon.plugins.module_utils.network.common.utils import to_list, ComplexList
+from ansible.module_utils.compat.version import LooseVersion
```

Comment:
This won't work with older ansible-core/-base/Ansible versions.

You can work around this by adding the files
https://github.com/ansible-collections/community.crypto/blob/main/plugins/module_utils/version.py
https://github.com/ansible-collections/community.crypto/blob/main/plugins/module_utils/_version.py
https://github.com/ansible-collections/community.crypto/blob/main/LICENSES/PSF-2.0.txt
to this collection as well, and changing the import to
```suggestion
from ansible_collections.community.routeros.plugins.module_utils.version import LooseVersion
```